lang/luajit/Makefile

```diff
@@ -0,0 +1,91 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luajit
+PKG_VERSION:=devel-2016-06-05-ce30766
```

Comment:
You can drop devel and only with 2016-06-05-githash
Think you can fresh to the latest commit?